libs/plugins/rudder-plugin-onetrust-consent-filter-react-native/android/src/main/java/com/rudderstack/plugin/reactnative/onetrust/OnetrustConsentFilterPlugin.kt

```
@@ -0,0 +1,113 @@
+            handleConsentDenied()
+        }
+        override fun onShowVendorList() {
+            handleConsentDenied()
+        }
+        override fun onHideVendorList() {
+            handleConsentDenied()
+        }
+        override fun onShowBanner(p0: OTUIDisplayReason?) {
+            handleConsentDenied()
+        }
+        override fun onVendorConfirmChoices() {}
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)